src/site/asciidoc/download.adoc

```diff
@@ -0,0 +1,229 @@
+|https://www.apache.org/dist/logging/log4j/{project-version}/apache-log4j-{project-version}-bin.zip.asc[apache-log4j-{project-version}-bin.zip.asc]
+
+|Apache Log4j source (tar.gz)
+|https://www.apache.org/dyn/closer.lua/logging/log4j/{project-version}/apache-log4j-{project-version}-src.tar.gz[apache-log4j-{project-version}-src.tar.gz]
+|https://www.apache.org/dist/logging/log4j/{project-version}/apache-log4j-{project-version}-src.tar.gz.sha512[apache-log4j-{project-version}-src.tar.gz.sha512]
+|https://www.apache.org/dist/logging/log4j/{project-version}/apache-log4j-{project-version}-src.tar.gz.asc[apache-log4j-{project-version}-src.tar.gz.asc]
+
+|Apache Log4j source (zip)
+|https://www.apache.org/dyn/closer.lua/logging/log4j/{project-version}/apache-log4j-{project-version}-src.zip[apache-log4j-{project-version}-src.zip]
+|https://www.apache.org/dist/logging/log4j/{project-version}/apache-log4j-{project-version}-src.zip.sha512[apache-log4j-{project-version}-src.zip.sha512]
+|https://www.apache.org/dist/logging/log4j/{project-version}/apache-log4j-{project-version}-src.zip.asc[apache-log4j-{project-version}-src.zip.asc]
+|===
+
+It is essential that you verify the integrity of the downloaded files
+using the PGP or SHA512 signatures. Please read
+https://httpd.apache.org/dev/verification.html[Verifying Apache HTTP
+Server Releases] for more information on why you should verify our
+releases.
+
+The PGP signatures can be verified using PGP or GPG. First download the
+https://downloads.apache.org/logging/KEYS[KEYS] as well as the asc
+signature file for the relevant distribution. Make sure you get these
+files from the https://www.apache.org/dist/logging/[main distribution
+directory], rather than from a mirror. Then verify the signatures using
+
+[source,sh,subs=attributes]
+----
+gpg --import KEYS
+gpg --verify apache-log4j-{project-version}-bin.zip.asc
+----
+
+Alternatively, you can verify the SHA512 signature on the files. A unix
+program called sha512sum is included in many unix distributions.
+
+[#requirements]
+== Requirements
+
+The Apache Log4j runtime requires a minimum of Java 17.
+See the link:http://logging.apache.org/2.x[Log4j 2 website] for the latest releases supporting Java 6, 7, and 8.
+
+== Previous Releases
+
+All previous releases of Apache log4j can be found in the
+https://archive.apache.org/dist/logging/log4j/[ASF archive repository]. Of course, all releases are availble for
+use as dependencies from the https://search.maven.org/search?q=org.apache.logging.log4j[Maven Central Repository]
+
+== Using Apache Log4j on your classpath
+
+To use Log4j in your application make sure that the API, Plugins, and Core
+jars are in the application’s classpath. Add the dependencies listed
+below to your classpath.
+
+* log4j-api-{project-version}.jar
+* log4j-plugins-{project-version}.jar
+* log4j-core-{project-version}.jar
```

Comment:
@ppkarwasz, is this list complete? Do we need to mention of your new `log4j-config-*` modules here?

Reply:
this list was compiled from the old docs, I suggest this should not be treated as part of this PR but a general fix outside the scope of it